fiware-region-sanity-tests/resources/settings.json

```diff
@@ -11,6 +11,10 @@
     "test_configuration": {
         "phonehome_endpoint": ""
     },
+    "swift_configuration": {
```

Comment:
I think it is better to put this config inside the _"test_configuration"_ property.